tests/test_SequenceManager.cc

```diff
@@ -529,3 +533,266 @@ TEST_CASE("SequenceManager: store_sequence() & load_sequence() - Empty sequence"
         REQUIRE(seq_deserialized.empty());
     }
 }
+
+
+TEST_CASE("SequenceManager: git repository", "[SequenceManager]")
+{
+
+    SECTION("Create and store sequence")
+    {
+        // clean surrounding
+        std::filesystem::remove_all(temp_dir);
```

Comment:
This removes the entire `unit_test_files` directory, and hence all the leftovers of other tests that ran before. But AFAIR we wanted to keep these leftovers so they could be examined after running the tests. It might be better to run this test in a subdirectory and only remove that one.